### ISSUE TITLE
Selection.onSelectionChanged callback should take ISelection as an arg (related to #9882)

### ISSUE THREAD

**Issue by flavomulti** [OfficeDev/office-ui-fabric-react#10717]
The fix described in #9882 no longer seems to works so there is no way to get access to the selection object inside the onSelectionChanged constructor argument, which makes it much less useful.

I the fix does not work, can we just have the callback in the constructor take an ISelection parameter? I think that's the simplest approach here.

```typescript
const s = new Selection({onSelectionChanged: (s: ISelection) => doSomething(s.getSelection())})
```
Without it, we are always pushing the Selection object up one level where the callback can be inserted at that level or doing complicated extra code, and we are forced to always pass it down. But there is no reason for extra code or other parameter passing with the fix above.

**Comment by khmakoto** [comment]
Hi @flavomulti, thanks for filing this issue! Can you elaborate on why the fix described in #9882 has stopped working? That could help further inform our decisions regarding this request. Thanks!

**Comment by flavomulti** [comment]
Hmm...this may be pilot error as it looks like the event publishing code is still there. I'm going to close for now.
